RecoPixelVertexing/PixelLowPtUtilities/test/ClusterShapeExtractor.cc

```diff
@@ -90,7 +90,8 @@ class ClusterShapeExtractor : public edm::EDAnalyzer
    bool hasSimHits;
    bool hasRecTracks;
 
-   edm::EDGetTokenT<SiPixelClusterShapeCache> theClusterShapeCacheToken;
+   edm::EDGetTokenT<edmNew::DetSetVector<SiPixelRecHit>> pixelRecHits_token;
+   edm::EDGetTokenT<SiPixelClusterShapeCache> clusterShapeCache_token;
```

Comment:
"the" in "theClusterShapeCache" was originally meant to denote the data member (as opposed to a local scope variable)